\hypertarget{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_successful}{}\section{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Successful Class Reference}
\label{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_successful}\index{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Successful@{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Successful}}


Inherits \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_exception}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Exception}.



Inherited by \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_accepted}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Accepted}, \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_created}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Created}, \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_no_content}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+No\+Content}, \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_non_authoritative_information}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Non\+Authoritative\+Information}, \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_ok}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Ok}, \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_partial_content}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Partial\+Content}, and \hyperlink{classaiohttp_1_1web__exceptions_1_1_h_t_t_p_reset_content}{aiohttp.\+web\+\_\+exceptions.\+H\+T\+T\+P\+Reset\+Content}.

\subsection*{Additional Inherited Members}


\subsection{Detailed Description}
\begin{DoxyVerb}Base class for exceptions with status codes in the 200s.\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/lib/python3.\+6/site-\/packages/aiohttp/web\+\_\+exceptions.\+py\end{DoxyCompactItemize}
